src/utils/defaultSettings.js

```diff
@@ -23,6 +23,8 @@ settingsProvider.setInitialValue('settings-miniplayer-show-task', false) // hide
 
 settingsProvider.setInitialValue('settings-lyrics-provider', '1') // OVH
 
+settingsProvider.setInitialValue('settings-miniplayer-show-top', false) // show on top always
```

Comment:
This supposed to be `settings-miniplayer-always-top`?

Reply:
No actually, but looking into this there is no initial setting for `settings-miniplayer-always-top`. I guess I'll fix that